proto/proto/store/rpc.proto

```diff
@@ -259,6 +262,50 @@ message SyncStateResponse {
     repeated note.NoteSyncRecord notes = 7;
 }
 
+// SYNC ACCOUNT VAULT
+// ================================================================================================
+
+// Account vault synchronization request.
+//
+// Allows clients to sync asset values for specific public accounts within a block range.
+message SyncAccountVaultRequest {
+    // Block number from which to start synchronizing.
+    fixed32 block_from = 1;
+    
+    /// Block number up to which to sync. If not specified, syncs up to the latest block.
+    ///
+    /// If specified, this block must be close to the chain tip (i.e., within 30 blocks),
+    /// otherwise an error will be returned.
+    optional fixed32 block_to = 2;
+    
+    // Account for which we want to sync storage maps.
+    account.AccountId account_id = 3;
+}
+
+message SyncAccountVaultResponse {
+    // The block number of the last row included in this response.
+    // For chunked responses, this may be less than request.block_to.
+    fixed32 block_num = 1;
+
+    // Chain tip at the moment of the request.
+    fixed32 chain_tip = 2;
+    
+    // List of asset updates for the account.
+    repeated AccountVaultUpdate updates = 3;
+}
+
+message AccountVaultUpdate {
+    // Vault key asscoiated with the asset.
+    primitives.Digest vault_key = 1;
+
+    // Serialized asset.
+    // If not present, the asset was removed from the vault.
+    optional bytes asset = 2; // TODO: see if it's worth making a message for assets
```

Comment:
Yes, let's make a separate message for `Asset`. For now, it could be just `Digest`.